week4-functions/index.html

```diff
@@ -0,0 +1,109 @@
+---
+layout: default
+permalink: /week4-functions/
+---
+    <h2>Quiz 4: Functions</h2>
+    <p>
+      <img src="ice_cream_function.png" width=200 height=200></iframe>
+    </br>
+      1. Recreate the above image by calling <code>function iceCream()</code> in a nested <code>for</code> loop.
```

Comment:
I think the nested loop is makes this a bit too difficult, it might be better with a single loop to emphasize the function aspect.